src/main/kotlin/org/jitsi/rtp/ByteArrayBufferImpl.java

```diff
@@ -152,6 +153,17 @@ public void shrink(int len)
             this.length = 0;
     }
 
+    /**
+     * Creates a clone of this buffer. The underlying byte[] has the same size
+     * as our byte[], but we only copy the data that this {@link ByteArrayBufferImpl}
+     * represents.
+     */
+    protected byte[] cloneBuffer()
```

Comment:
where did this and `isInvalid` used to be?

Reply:
Oh, `cloneBuffer()` was moved or removed in master, I just didn't notice while rebasing. I'll remove it.

`isInvalid` is new to `jitsi-rtp`. The libjitsi `ByteArrayBuffer` interface uses it, so I had to add it to the `ByteArrayBuffer` interface in jitsi-utils.